### DISCUSSION TITLE
Auto assign template

### DISCUSSION THREAD

**Discussion by rioshaz** [YetiForceCompany/YetiForceCRM#14488] [How to]
how to and where can i create a auto assign template ?
how does this work ?
![image](https://user-images.githubusercontent.com/54880467/113110074-48099e80-9224-11eb-9b4c-22219616a312.png)

**Comment by mariuszkrzaczkowski** [comment]
https://gitdeveloper.yetiforce.com/index.php?module=AutomaticAssignment&view=List&parent=Settings
![image](https://user-images.githubusercontent.com/10198654/113110354-42519000-9207-11eb-9312-b7f00cf2a988.png)

**Comment by rioshaz** [comment]
thanks buddy but there is no way to select a picklist when making a auto-assign,,, only fields with text are shown... but status of a oder shall define where it goes...

![image](https://user-images.githubusercontent.com/54880467/113113254-bb60df80-9227-11eb-8970-0cbb4853a28a.png)


i want: 
when an order is created it should go  another team and in a sequence like person1, person2, person3 and repeat... is that possible ?
also the record creator should be able to see different stage the order is in... but cant edit anything...

status picklist shall define which department order goes

any help on how to acheieved this ?

**Reply by rioshaz** [reply]
its impossible for me to tell the system order no.  its generated by system, everytime when a new order is added..... pls share u expert opinion.

also what about if the auto assigning user is in off or is away for few days,,

**Reply by rioshaz** [reply]
> thanks buddy but there is no way to select a picklist when making a auto-assign,,, only fields with text are shown... but status of a oder shall define where it goes...
> 
> ![image](https://user-images.githubusercontent.com/54880467/113113254-bb60df80-9227-11eb-8970-0cbb4853a28a.png)
> 
> i want:
> when an order is created it should go another team and in a sequence like person1, person2, person3 and repeat... is that possible ?
> also the record creator should be able to see different stage the order is in... but cant edit anything...
> 
> status picklist shall define which department order goes
> 
> any help on how to acheieved this ?

i think i found solution for this.. please help me with other questions, if u can

**Reply by rioshaz** [reply]
> its impossible for me to tell the system order no. its generated by system, everytime when a new order is added..... pls share u expert opinion.
> 
> also what about if the auto assigning user is in off or is away for few days,,

@mariuszkrzaczkowski can u help ?
